www/friendly.html.pm

```diff
@@ -11,7 +11,20 @@ The Racket community aims to improve the world through programming. It started w
 
 Racket project management wants to encourage and preserve this open exchange of ideas, which requires an environment that enables all to participate without fear of personal harassment. We define ◊em{harassment} to include specific unacceptable factors and behaviors listed in ◊link["https://www.acm.org/about-acm/policy-against-harassment"]{the ACM's policy against harassment}. Unacceptable behavior will not be tolerated.
 
-Anyone witnessing or subject to unacceptable behavior should notify ◊link["team.html#management"]{Racket project management}, which can be reached at ◊link["mailto:management@racket-lang.org"]{management@racket-lang.org}.
+Anyone witnessing or subject to unacceptable behaviour should notify the moderation team or Racket Project management.
+
+Community moderation team:
+
+◊link["https://github.com/countvajhula"]{Siddhartha Kasivajhula}
+
+◊link["https://github.com/jackfirth"]{Jack Firth}
+
+◊link["https://github.com/samdphillips"]{Sam Phillips}
+
+◊link["https://github.com/spdegabrielle"]{Stephen De Gabrielle}
+
+
+◊link["team.html#management"]{Racket project management} can be reached at ◊link["mailto:management@racket-lang.org"]{management@racket-lang.org}.
```

Comment:
Re: possible confusion with project management and community moderators, how about retaining the references to "project management" in these changes, but rewording the paragraph above to something like:

_In order to encourage and preserve this open exchange of ideas, we seek to foster an environment that enables all to participate without fear of personal harassment. We define ◊em{harassment} to include specific unacceptable factors and behaviors listed in ◊link["https://www.acm.org/about-acm/policy-against-harassment"]{the ACM's policy against harassment}. Unacceptable behavior will not be tolerated._

That way the only references to project management make it clear that they are another recourse and not the subject here.

Reply:
I like the suggested text, but I belive the risk of confusion is low.

Can we review this textual change as a follow-up PR, once this one is approved?

**Racket project management wants** 
_to encourage and preserve this open exchange of ideas,_ 
**which requires an** 
_environment that enables all to participate without fear of personal harassment._

**In order** 
_to encourage and preserve this open exchange of ideas,_ 
**we seek to foster an** 
_environment that enables all to participate without fear of personal harassment._

Reply:
I think that obfuscating the email addresses in this is no barrier to the spammers, only a (mild) barrier to the legit folks who want to contact the moderation team. If it were me, I'd put in `mailto:` links.

Reply:
@countvajhula created a new issue so this feedback isn't lost and can be addressed later #239